core/src/test/resources/assets/sql/missing_last_semicolon.sql

```diff
@@ -1,10 +0,0 @@
--- We ommit the trailing semicolon of the last statement to test whether the parser detects that.
-CREATE TABLE 'testTable'
```

Comment:
more readable as files though - maybe raise an issue on the repo to revert this when we can